indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
+        {% if action == 'withdrawn' %}
+            {% trans name=judge_name -%}
+                <strong>{{ name }}</strong> withdrew this abstract
```

Comment:
But I don't think it's a big deal if for now they always see it's been the submitter. After all, it will be at the submitter's request.